modules/trusted-firmware-m/tfm_boards/CMakeLists.txt

```diff
@@ -79,33 +79,41 @@ if (${TFM_PARTITION_CRYPTO})
     )
   endif()
 
+
+  if (${CONFIG_HW_UNIQUE_KEY})
+    target_compile_definitions(platform_s PUBLIC
+      CONFIG_HW_UNIQUE_KEY
+      $<$<BOOL:${CONFIG_HW_UNIQUE_KEY_RANDOM}>:CONFIG_HW_UNIQUE_KEY_RANDOM>)
+
+    # nrf54l15 uses the PSA headers in the hw_unique_key_cracen.c file, which means
+    # that we need to make sure that the nrf_security PSA headers are included
+    # before any other PSA-related headers.
+    if((NRF_SOC_VARIANT STREQUAL nrf54l15) OR (target STREQUAL nrf54l15))
+       target_link_libraries(platform_crypto_keys
+        PRIVATE
+        psa_crypto_library_config
+       )
+
+       target_sources(platform_crypto_keys
+         PRIVATE
+          ${ZEPHYR_NRF_MODULE_DIR}/lib/hw_unique_key/hw_unique_key_cracen.c
+       )
+    else()
+       target_sources(platform_s
+         PRIVATE
+           ${ZEPHYR_NRF_MODULE_DIR}/lib/hw_unique_key/hw_unique_key.c
+           ${ZEPHYR_NRF_MODULE_DIR}/lib/hw_unique_key/hw_unique_key_kmu.c
```

Comment:
align the filenames with PRIVATE

Reply:
I saw that, it is like that in all the file though. I will try to fix it file wide then.

Reply:
Can we keep formatting refactors for another time?

Reply:
I just updated the whole file, please double check.